packages/db/src/core/cli/migration-queries.ts

```diff
@@ -40,12 +40,23 @@ const genTempTableName = customAlphabet('abcdefghijklmnopqrstuvwxyz', 10);
 export async function getMigrationQueries({
 	oldSnapshot,
 	newSnapshot,
+	reset = false
 }: {
 	oldSnapshot: DBSnapshot;
 	newSnapshot: DBSnapshot;
+	reset?: boolean;
 }): Promise<{ queries: string[]; confirmations: string[] }> {
-	const queries: string[] = [];
+	let queries: string[] = [];
```

Comment:
nit: can we keep this as a const, and instead of overwriting the array do `queries.push(...getDropTableQueriesForSnapshot(curSnapshot))`? My fear is that future devs blow away existing queries accidentally by re-assigning